pkgs/racket-test-core/tests/racket/portlib.rktl

```diff
@@ -14,14 +14,29 @@
   (test '(#\h #\e #\l #\l #\o #\return #\newline #\t #\h #\e #\r #\e)
 	port->list read-char (p))
   (test "hello\r\nthere" port->string (p))
+  (test "hello\r\nthere" port->string/close (p))
```

Comment:
Is `port->string/close` defined here? (I was waiting for the build to finish to see, but might as well ask)